NDTensors/ext/NDTensorsGPUArraysCoreExt/contract.jl

```diff
@@ -0,0 +1,48 @@
+using Adapt: adapt
+using GPUArraysCore: AbstractGPUArray
+using NDTensors: NDTensors, DenseTensor, DiagTensor, contract!, dense, inds, Tensor
```

Comment:
Please sort alphabetically with capitalized names grouped first (specifically, `Tensor` should be grouped with the other types).

Reply:
Oh thanks I wasn't thinking about types. It should be fixed now. Thanks!